src/PKSim.Core/Commands/SetSpeciesInCompoundProcessCommand.cs

```diff
@@ -35,6 +35,9 @@ public override void Execute(IExecutionContext context)
 
             Add(new SetParameterValueCommand(parameter, individualParameter.Value) {Visible = false});
       
+            //Ensure that parameter is not marked as default as it was changed from default value defined in DB
+            Add(new SetParameterDefaultStateCommand(parameter, isDefault: false) { Visible = false });
```

Comment:
My idea was to ensure that species dependent parameters were also set as changed. But it seems that you do not want that?

Reply:
Note: This is not requried if we use the default species (see below)